copilot/rails/manifest.yml

```diff
@@ -30,14 +33,19 @@ image:
   # Port exposed through your container to route traffic to it.
   port: 3000
 
-command: bundle exec pumactl start
-
 cpu: 256       # Number of CPU units for the task.
 memory: 512    # Amount of memory in MiB used by the task.
 count: 1       # Number of tasks that should be running in your service.
 
 exec: true     # Enable running commands in your container.
 
+sidecars:
+  nginx:
+    image: 353381651656.dkr.ecr.ap-northeast-1.amazonaws.com/chronos/nginx:latest
+    port: 80
+    variables:
+      RAILS_HOST: http://localhost:3000
```

Comment:
Containers that belonging to same task can communicate over `localhost`.
https://github.com/aws/copilot-cli/issues/2370